local-volume/provisioner/pkg/discovery/discovery_test.go

```diff
@@ -331,17 +331,26 @@ func findSCName(t *testing.T, targetDir string, test *testConfig) string {
 }
 
```

Comment:
@NickrenREN - can we add a test or tests for UseAlphaAPI?

Reply:
yes, will do

Reply:
done